Add exclude open prs (#14)
beatlabs/delete-old-branches-action: README.md

Additional branches can be excluded using the `extra_protected_branch_regex` variable (see example below).
Similarly, certain tags can be excluded using the `extra_protected_tag_regex` variable.

> The default behaviour is to exclude Github protected branches as well as branches with open pull requests. Additional branches can be excluded using the `extra_protected_branch_regex` variable.  Branches with open pull requests can be ignored as well (see example below).  

Reviewer: We are mentioning above the github protected branches. So, I suggest the following here:
```suggestion
Also, branches with open pull requests are being ignored by default (see example below).  
```
